gufe/tests/test_proteincomponent.py

```diff
@@ -239,6 +239,22 @@ def test_to_openmm_topology(self, in_pdb_path):
         assert openmm_top.getNumChains() == gufe_openmm_top.getNumChains()
         assert openmm_top.getNumResidues() == gufe_openmm_top.getNumResidues()
         assert openmm_top.getNumChains() == gufe_openmm_top.getNumChains()
+        # Make sure bond.order is the expected type int or None
+        assert all(isinstance(bond.order, (int, type(None))) for bond in openmm_top.bonds())
```

Comment:
@mikemhenry  Maybe we should just add a new line that tests for the types of `bond.type` here and it should be enough as regression tests. As in, if we ever go back to NOT having the correct types, these two lines in this test will fail.

Reply:
However, adding check for `bond.type` is a bit ugly, since it depends on the custom types/objects defined in openmm (`Single`, `Double`, etc.)